wisetools.py

```diff
@@ -232,7 +232,7 @@ def scaleSample(sample, fromSize, toSize):
 		newLen = int(np.ceil(len(chromData)/float(scale)))
 		scaledChrom = np.zeros(newLen, dtype=np.int32)
 		for i in range(newLen):
-			scaledChrom[i] = np_sum(chromData[i*scale:i*scale+scale])
+			scaledChrom[i] = np_sum(chromData[int(i*scale):int(i*scale+scale)])
```

Comment:
I'm not sure if this change makes much sense, just before this edit there is this:
```
if toSize == 0 or fromSize == 0 or toSize < fromSize or toSize % fromSize > 0:
	print 'ERROR: Impossible binsize scaling requested:', fromSize, 'to', toSize
	exit(1)
```

Basically I tried to make sure the scaling was sensible, which is only the case if it is an exact multiple. Putting a float as `scale` would result in undesired effects as that would require splitting bins into sub bins, at best one could split a bins value into two sub bins, providing no improvements in results as far as I can imagine.

Wouldn't a better fix be a proper error check rather than forcing a float to work for `scale`?